source/_integrations/recorder.markdown

```diff
@@ -392,6 +392,18 @@ sudo apt-get install postgresql-server-dev-X.Y
 pip3 install psycopg2
 ```
 
+If this doesn't work you may have to install the psycopg2 package as part of the same virtual environment that runs Home Assistant
+```bash
+pip3 uninstall psycopg2
+sudo systemctl stop home-assistant@homeassistant.service
+sudo -u homeassistant -H -s
+source /srv/homeassistant/bin/activate
+pip3 install psycopg2
+```
+
+Then hit Control-D to exit the virtual environment.
+
+
```

Comment:
I think it's better to document that if Home Assistant is installed in a virtual environment, the additional libraries should be installed in the same virtual environment. This is also the case for mysql / mariadb.

Reply:
I made changes based on your suggestion, but didn't notice that you had already made / suggested a change. I'm reasonably familiar with version control but not so much with HA or git. Apologies if I haven't done this quite correctly, and if you are able to fix it up please go ahead!